### DISCUSSION TITLE
Is this phenomenon normal?

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#8737] [Q&A]
When I delete 20% of the training nodes in the CORA dataset, I find that the accuracy is 1% higher than when I didn't delete it, and only when I delete more than 50% of the training nodes, the accuracy will drop significantly.Why is this happening? it is because removing some nodes makes the model more robust?

**Comment by rusty1s** [comment]
That's a cool finding. Does that also hold true when training the models multiple times (e.g., 100 times) and compare the mean accuracy over the models?

**Reply by Chrisable** [reply]
yes

**Reply by Chrisable** [reply]
i just use the GCN model

**Reply by rusty1s** [reply]
Do you mind sharing your example?

**Reply by Chrisable** [reply]
I used your GCN example to train CORA, but used the subgraph to randomly remove some nodes before training

**Reply by Chrisable** [reply]
Everything else is consistent

**Reply by rusty1s** [reply]
Yeah, can you share the code you used to generate subgraphs?

**Reply by Chrisable** [reply]
Yes, please ask what is your email address?

**Reply by rusty1s** [reply]
You can look it up on my GitHub profile. Looking forward to hear from you.
